_layouts/default.html

```diff
@@ -112,16 +112,17 @@ <h3>Subscribe to announcements</h3>
           <div>
             <h3>Exchanges</h3>
             <ul class="quick-links">
-              <li><a href="https://bittrex.com/Market/Index?MarketName=BTC-AEON" rel="nofollow">Bittrex</a> [aeon/btc]</li>
-              <li><a href="https://hitbtc.com/exchange/AEON-to-BTC" rel="nofollow">HitBTC</a> [aeon/btc]</li>
-              <li><a href="https://tradeogre.com/exchange/BTC-AEON" rel="nofollow">TradeOgre</a> [aeon/btc]</li>
-              <li><a href="https://bitliber.com/markets/aeon-btc" rel="nofollow">Bitliber</a> [aeon/btc]</li>
-              <li><a href="https://bitliber.com/markets/aeon-xmr" rel="nofollow">Bitliber</a> [aeon/xmr]</li>
-              <li><a href="https://letsdocoinz.com/AEON-BTC" rel="nofollow">LetsDoCoinz</a> [aeon/btc]</li>
-              <li><a href="https://changenow.io/?from=btc&to=aeon&amount=0.01" rel="nofollow">ChangeNOW</a> [aeon/any]</li>
-              <li><a href="https://simpleswap.io" rel="nofollow">SimpleSwap</a> [aeon/any]</li>
-              <li><a href="https://Xchange.me" rel="nofollow">Xchange</a> [any->aeon]</li>
+              <li><a href="https://bittrex.com/Market/Index?MarketName=BTC-AEON" rel="nofollow">Bittrex</a> [aeon<=>btc]</li>
+              <li><a href="https://hitbtc.com/exchange/AEON-to-BTC" rel="nofollow">HitBTC</a> [aeon<=>btc]</li>
+              <li><a href="https://tradeogre.com/exchange/BTC-AEON" rel="nofollow">TradeOgre</a> [aeon<=>btc]</li>
+              <li><a href="https://bitliber.com/markets/aeon-btc" rel="nofollow">Bitliber</a> [aeon<=>btc]</li>
+              <li><a href="https://bitliber.com/markets/aeon-xmr" rel="nofollow">Bitliber</a> [aeon<=>xmr]</li>
+              <li><a href="https://letsdocoinz.com/AEON-BTC" rel="nofollow">LetsDoCoinz</a> [aeon<=>btc]</li>
+              <li><a href="https://changenow.io/?from=btc&to=aeon&amount=0.01" rel="nofollow">ChangeNOW</a> [aeon<=>any]</li>
+              <li><a href="https://simpleswap.io" rel="nofollow">SimpleSwap</a> [aeon<=>any]</li>
+              <li><a href="https://Xchange.me" rel="nofollow">Xchange</a> [any=>aeon]</li>
               <li><a href="https://bisq.network/markets/?currency=aeon_btc" rel="nofollow">Bisq</a> [aeon<=>btc]</li>
+              <li><a href="https://aeonapi.com" rel="nofollow">AeonAPI</a> [Aeon=>BTC]</li>
```

Comment:
Maybe uncapitalize the currency symbols like the rest of the lines?

Reply:
will you squash commits if I make another commit?

Reply:
No, squashing needs to be done by you because you’re the author of this PR. You can easily do it by `git rebase -i HEAD^^`